catalog/imaging/imaging-actionprovider-chip/src/main/java/org/codice/alliance/imaging/chip/actionprovider/ImagingChipActionProvider.java

```diff
@@ -56,16 +70,20 @@
             URL url;
             String chipPath = null;
             try {
-                chipPath =
-                        SystemBaseUrl.getBaseUrl() + PATH + "?id=" + metacard.getId() + "&source="
-                                + metacard.getSourceId();
+                chipPath = String.format("%1s%2s?id=%3s&source=%4s",
```

Comment:
This is fine, but for future reference you don't need to specify the position of the tokens. `%s%s?id=%s&source=%s` would work fine.